pyaedt/modules/report_templates.py

```diff
@@ -205,10 +205,44 @@ class Trace(object):
     def __init__(self, report_setup, trace_name):
         self._oreport_setup = report_setup
         self.trace_name = trace_name
+        self._trace_name_real = None
         self.LINESTYLE = LineStyle()
         self.TRACETYPE = TraceType()
         self.SYMBOLSTYLE = SymbolStyle()
 
+    @property
+    def trace_name_real(self):
```

Comment:
please add docstring

Reply:
@maxcapodi78 @gmalinve I think self.trace_name should be the name of the trace for instance "dB(S1:1)" and we should change the current self.trace_name, because it contains the following info: Report name + trace name + variation, for instance: "MyReport:dB(S1:1):$v=1,s=2", I think it is confusing. 

Then self.trace_name should be the name of the trace, and the other property could be called self.trace_aedt_name or something like this...

Reply:
@maxcapodi78 @Samuelopez-ansys I'd change as:
- self.trace_name to be name of trace, i.e: "dB(S(1:1,1:1))".
- and something like self.trace_aedt_name what we have now for self.trace_name ("MyReport:dB(S1:1):$v=1,s=2")
Could it be a bit more clear?